qemu/tests/ept_test.py

```diff
@@ -0,0 +1,40 @@
+from avocado.utils import process, cpu
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    ept test:
+    1) Turn off ept on host
+    2) Check kvm_intel parameter
+    3) Launch a guest
+    3) Check no error in guest
+    4) Restore env, turn on ept
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    if cpu.get_cpu_vendor_name() != 'intel':
+        test.cancel("This test is supposed to run on Intel host")
+
+    unload_cmd = params["unload_cmd"]
+    load_cmd = params["load_cmd"]
+    test_cmd = params["test_cmd"]
+    process.system(unload_cmd)
+    process.system(load_cmd % "0")
+    process.system(test_cmd)
```

Comment:
@huangyum  Only `cat` ?  why do not check the content of `vmentry_l1d_flush` ? `EPT disabled` when `ept=0`

Reply:
Yes, they are the steps to trigger a system crash before. You can refer to the bug linked to polarion case. And the content may vary on different hosts.

Reply:
Got it, thanks!